packages/homepage/src/pages/podcasts/codesandbox-podcast.js

```diff
@@ -95,35 +108,34 @@ const VersionOne = ({
                 </section>
               </Episode>
             </Link>
-          ))}
+          ))} */}
       </Episodes>
       <PodcastLinks {...info.links} name={info.name} />
     </PageContainer>
   </Layout>
 );
 
-export const query = graphql`
-  {
-    allMarkdownRemark(
-      filter: { fileAbsolutePath: { regex: "/podcasts/codesandbox-podcast/" } }
-      sort: { fields: id, order: DESC }
-    ) {
-      edges {
-        node {
-          id
-          frontmatter {
-            description
-            episodeNumber
-            title
-            slug
-            image {
-              publicURL
-            }
-          }
-        }
-      }
-    }
-  }
-`;
+// export const query = graphql`
```

Comment:
Why did you remove this? You gonna have to put it back when there are episodes

Reply:
Let me know if you want to have a call

Reply:
Hi Sara, 
Yeah, I had an issue when I deleted the first episode's MD  files. All worked well locally, but it would not build the PR. 
Scott could debug it and discovered that something went wrong because I deleted the MD files. 
That's why I have tried to turn it off. 

The goal is to have the pages available with the trailers but no episodes yet (next week). 
And as soon as the episodes are released to show the episodes. 

I will uncomment the parts as soon as the first episode is up. #hackhack